.cypress/cypress/integration/peterborough.js

```diff
@@ -18,7 +20,7 @@ describe('new report form', function() {
     cy.get('.js-post-category-messages:visible').should('contain', 'Please phone customer services to report this problem.');
     cy.get('.js-reporting-page--next:visible').should('be.disabled');
     cy.get('#form_emergency').select('no');
-    cy.get('.js-post-category-messages:visible').should('not.contain', 'Please phone customer services to report this problem.');
+    cy.get('.js-post-category-messages:hidden').should('not.contain', 'Please phone customer services to report this problem.');
```

Comment:
I didn't follow the reason for this change

Reply:
I'm not sure either, I've reverted it.